test/Microsoft.ApplicationInsights.AspNetCore.Tests/HostingDiagnosticListenerTest.cs

```diff
@@ -358,15 +355,12 @@ public void OnEndRequestSetsRequestNameToMethodAndPath(bool isAspNetCore2)
         {
             HttpContext context = CreateContext(HttpRequestScheme, HttpRequestHost, "/Test", method: "GET");
             TelemetryConfiguration config = TelemetryConfiguration.CreateDefault();
```

Comment:
this is an unrelated change. conditional app is tested separately from this test. Dont want to pollute this test with unrelated change.